applications/nrf_desktop/src/util/Kconfig

```diff
@@ -21,4 +21,18 @@ config DESKTOP_ADV_PROV_UUID16_ALL
 	  Adds all UUID16 to the advertising payload if used Bluetooth local
 	  identity has no bond.
 
+config DESKTOP_DFU_LOCK
+	bool "DFU lock module"
```

Comment:
I wonder if we should not switch from implying the option in DFU transports to setting the option to `y` by default here if both transports are enabled (thanks to that we would keep the DFU lock module disabled when not needed without need of explicitly disabling it in most of the configurations). In such case, maybe we could  actually accept that the module would know about the clients (both DFU lock and clients are part of nRF Desktop application).